### ISSUE TITLE
Add CircleCI Tests

### ISSUE THREAD

**Issue by mathomp4** [GEOS-ESM/GEOSldas#285]
A question mainly for @gmao-rreichle. Would you like me to add CircleCI build tests (on PRs a la GEOSgcm, etc.) to GEOSldas? I think it would be fairly simple, and now that we have access to bigger resources there, build tests will probably be sub-10 minutes.

My only need would be to know exactly what you want checked out. For example, on GEOSgcm we currently do:
```
mepo clone
mepo develop GEOSgcm_App GEOSgcm_GridComp
```
I'm guessing GEOSgcm_App isn't needed, but I can change to any develop branch. I could also do like GEOSgcm_GridComp where we have:
```
mepo checkout-if-exists ${CIRCLE_BRANCH}
```
where if someone has a branch, say, here and in GEOSgcm_GridComp with the same name, we can grab both at the same time.

**Comment by gmao-rreichle** [comment]
Thanks, @mathomp4 , for taking a look at this.  I may not be qualified to answer the question, but here are a couple of thoughts:
Generally, yes, I think adding CircleCI build tests for GEOSldas PRs is a good idea if we can afford it. 
Re. what needs to be checked out: If this is for a PR, doesn't the branch specified in the PR need testing? 
That is, if I'm creating a PR off GEOSldas branch "rolfs_new_stuff", it should come with complete info about what branches/tags from other repos are needed (via "components.yaml" and "Externals.cfg").  But I haven't really made the switch to "mepo" and I don't really know what I'm talking about.
I'll ask @weiyuan-jiang to enlighten me when I meet with him on Wed morning.

**Comment by gmao-rreichle** [comment]
@mathomp4 : After more consultation with @weiyuan-jiang , we feel that it'd be ok to add the test of the branch associated with PR as specified in Externals.cfg (or components.yaml, which we will keep consistent with Externals.cfg), but it's not a critical need.
Testing other variants (ie, using develop branches for something that is specified as a tag in Externals.cfg) would probably add more confusion than be helpful. 
We'll leave it up to you to add the CircleCI build test at this time or shelf the issue for another day.
Thanks!

**Comment by mathomp4** [comment]
Okay. I think this might be possible now that GEOSldas is using mepo by default. I might look at adding the CI Build.

**Comment by gmao-rreichle** [comment]
Addressed in pull request #413
